automobile/openstreetmap-importer.md

```diff
@@ -1,24 +1,37 @@
 ## OpenStreetMap importer
 
-In order to ease the creation of new environments for automobile simulations, a
-script has been created to convert maps from OpenStreetMap into Webots worlds.
+In order to ease the creation of new environments for automobile simulations,
+Webots worlds can be generated from OpenStreetMap maps using the importer script
+described here.
 
-You can download a map of any part of the world from
+You can download an OpenStreetMap map of any part of the world from
 [www.openstreetmap.org/export](http://www.openstreetmap.org/export) (do not use
 more than a few kilometers square if you want to be able to run your simulation
 in real-time). And then convert it using the script.
```

Comment:
For the sake of clarity, we could specifiy: "Save it as `myMap.osm`" and remove the: "And then convert it using the script" as conversion should happen after using `netconvert`, so it seems too early here.